parsely/build.gradle

```diff
@@ -22,15 +20,23 @@ android {
         exclude 'META-INF/LICENSE'
         exclude 'META-INF/NOTICE'
     }
+
+    publishing {
+        singleVariant('release') {
+            withSourcesJar()
+        }
+    }
+}
+
+ext {
+    PUBLISH_GROUP_ID = 'com.parsely'
+    PUBLISH_VERSION = '3.0.2'
+    PUBLISH_ARTIFACT_ID = 'parsely'
```

Comment:
1) Question (❓): Why did you chose this `build.gradle` file and location to put these publishing related information?
2) Suggestion (💡): If possible, consider moving them to a newly created file, maybe a root level `publication.properties` file, to match that of the `publication.gradle` file, something like the below. You will also note that I reorder them as well to have the version last and as such match its usage (`com.parsely:parsely:3.0.2`):

`publication.properties`
```
publish.groupId=com.parsely
publish.artifactId= parsely
publish.version=3.0.2

```

Reply:
That's for the extra clarification @wzieba !

Moving those to the `publication.gradle` seems a good choice too, 👍 from my side on that!

Reply:
I did yet another change: I've moved group and name of the artifact to publication script but I left `VERSION` in `build.gradle` of the module.

The reasons are:
1. Version is a coherent part of the build (doesn't matter if it's published)
2. It'll be easy to find it later as we'll need to update this version to release a new aar 🙂

I think with all of these we came to the best solution!

Reply:
@wzieba coolio, deal, I did check the changes you did and it all LGTM! 👍 💯